TypeError: cannot

We are using self-hosted runners and are unable to get this action to work.  It seems to run the setup step, but fails with any `terraform *` action:

This could be related to #13, but I didn't see this specific error, so I'm not sure how we are to troubleshoot it.
```
Run terraform fmt .
  terraform fmt .
  shell: /bin/bash -e {0}
  env:
    TERRAFORM_CLI_PATH: /home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3
    GITHUB_TOKEN: ***
/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1316
_a = fs.promises, exports.chmod = _a.chmod, exports.copyFile = _a.copyFile, exports.lstat = _a.lstat, exports.mkdir = _a.mkdir, exports.readdir = _a.readdir, exports.readlink = _a.readlink, exports.rename = _a.rename, exports.rmdir = _a.rmdir, exports.stat = _a.stat, exports.symlink = _a.symlink, exports.unlink = _a.unlink;
                                     ^

TypeError: Cannot read property 'chmod' of undefined
    at Object.672 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1316:38)
    at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
    at Object.1 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:65:16)
    at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
    at Object.950 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1552:12)
    at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
    at startup (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:38:19)
    at module.exports.1.P (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:42:18)
    at Object.<anonymous> (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:45:10)
    at Module._compile (module.js:652:30)
##[error]Process completed with exit code 1.
```

Same error at the init phase:

Run terraform init
/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:1316
_a = fs.promises, exports.chmod = _a.chmod, exports.copyFile = _a.copyFile, exports.lstat = _a.lstat, exports.mkdir = _a.mkdir, exports.readdir = _a.readdir, exports.readlink = _a.readlink, exports.rename = _a.rename, exports.rmdir = _a.rmdir, exports.stat = _a.stat, exports.symlink = _a.symlink, exports.unlink = _a.unlink;
                                     ^

TypeError: Cannot read property 'chmod' of undefined
    at Object.672 (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:1316:38)
    at __webpack_require__ (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:23:30)
    at Object.1 (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:65:16)
    at __webpack_require__ (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:23:30)
    at Object.950 (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:1552:12)
    at __webpack_require__ (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:23:30)
    at startup (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:38:19)
    at module.exports.1.P (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:42:18)
    at Object.<anonymous> (/home/marc/actions-runner/_work/_temp/a22c1279-976a-4fd2-91c0-45ecb0091b02/terraform:45:10)
    at Module._compile (module.js:652:30)
##[error]Process completed with exit code 1.

> We are using self-hosted runners and are unable to get this action to work. It seems to run the setup step, but fails with any `terraform *` action:
> 
> This could be related to #13, but I didn't see this specific error, so I'm not sure how we are to troubleshoot it.
> 
> ```
> Run terraform fmt .
>   terraform fmt .
>   shell: /bin/bash -e {0}
>   env:
>     TERRAFORM_CLI_PATH: /home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3
>     GITHUB_TOKEN: ***
> /home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1316
> _a = fs.promises, exports.chmod = _a.chmod, exports.copyFile = _a.copyFile, exports.lstat = _a.lstat, exports.mkdir = _a.mkdir, exports.readdir = _a.readdir, exports.readlink = _a.readlink, exports.rename = _a.rename, exports.rmdir = _a.rmdir, exports.stat = _a.stat, exports.symlink = _a.symlink, exports.unlink = _a.unlink;
>                                      ^
> 
> TypeError: Cannot read property 'chmod' of undefined
>     at Object.672 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1316:38)
>     at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
>     at Object.1 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:65:16)
>     at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
>     at Object.950 (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:1552:12)
>     at __webpack_require__ (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:23:30)
>     at startup (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:38:19)
>     at module.exports.1.P (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:42:18)
>     at Object.<anonymous> (/home/runner/_work/_temp/3a1a054c-4701-42f3-8dcd-5518d64d59b3/terraform:45:10)
>     at Module._compile (module.js:652:30)
> ##[error]Process completed with exit code 1.
> ```

Hi, 

For me the solution was to install additional packages. I've created a small script that installs the necessary software (it's far from perfect but it will do the job)

**#Update Ubuntu (done)**
sudo apt-get update

**#Install Unzip (done)**
sudo apt-get install -y unzip

**#Install Terraform (done)**
wget https://releases.hashicorp.com/terraform/0.12.28/terraform_0.12.28_linux_amd64.zip
sudo unzip "terraform_0.12.28_linux_amd64.zip" -d /usr/local/bin
rm -f "terraform_0.12.28_linux_amd64.zip"

**#Install go (done)**
sudo apt install golang

**#Install Grunt (done)**
curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-install | bash -s -- -ny -
~/n/bin/n lts
sudo npm install -g grunt gulp n parcel-bundler typescript
sudo npm install -g --save-dev webpack webpack-cli
sudo npm install -g npm
sudo rm -rf ~/n

**# Install Yarn repository and key**
sudo curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
sudo apt update
sudo apt-get install -y --no-install-recommends yarn

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ After you've used the action, subsequent steps in the same job can run arbitrary
 
 ## Usage
 
-This action can be run on `ubuntu-latest`, `windows-latest`, and `macos-latest` GitHub Actions runners. When running on `windows-latest` the shell should be set to Bash.
+This action can be run on `ubuntu-latest`, `windows-latest`, and `macos-latest` GitHub Actions runners. When running on `windows-latest` the shell should be set to Bash. When running on self-hosted GitHub Actions runners, NodeJS must be previously installed with the version specified in the [`action.yml`](https://github.com/hashicorp/setup-terraform/blob/main/action.yml).
 
 The default configuration installs the latest version of Terraform CLI and installs the wrapper script to wrap subsequent calls to the `terraform` binary:
 

